src/func.cpp

```diff
@@ -464,6 +464,8 @@ void Function::GenerateIR() {
 
                 llvm::Function *appFunction = llvm::Function::Create(ftype, linkage, functionName.c_str(), m->module);
                 appFunction->setDoesNotThrow();
+                if (g->calling_conv == CallingConv::x86_vectorcall)
```

Comment:
In this changeset the pattern "if if (g->calling_conv == CallingConv::x86_vectorcall) XXX->setCallingConv(llvm::CallingConv::X86_VectorCall);" appears at least 4 times. It worth adding a function to do that. This will also may check if the state of calling convention is not properly initialized"

Reply:
Done